test.sh

```diff
@@ -769,3 +769,11 @@ total_count=$(
   curl -sSfg -u $SK: $HOST/v1/charges?customer=$cus\&created[gt]=1588166306 \
   | grep -oE '"total_count": 6')
 [ -n "$total_count" ]
+
+# check starting_after parameter for the events
+# get the last event id
+event_id=$(curl -sSfg -u $SK: $HOST/v1/events?limit=10000 | grep -oE 'evt_\w+' | tail -n 1)
```

Comment:
I think the maximum allowed `limit` value is `100`: https://stripe.com/docs/api/pagination#pagination-limit. If we enforce it in the future, this test will fail. Let's use `100`?